prow/plugins.yaml

```diff
@@ -42,57 +42,170 @@ use_deprecated_2018_implicit_self_approve_default_migrate_before_july_2019: true
 use_deprecated_2018_review_acts_as_approve_default_migrate_before_july_2019: true
 
 approve:
-- repos:
-  - kubernetes/cloud-provider-aws
-  - kubernetes/cloud-provider-azure
-  - kubernetes/cluster-registry
-  - kubernetes/contrib
-  - kubernetes/dashboard
-  - kubernetes/dns
-  - kubernetes/enhancements
-  - kubernetes/examples
-  - kubernetes/federation
-  - kubernetes/gengo
-  - kubernetes/ingress-gce
-  - kubernetes/ingress-nginx
-  - kubernetes/klog
-  - kubernetes/kube-deploy
-  - kubernetes/kubeadm
-  - kubernetes/kubectl
-  - kubernetes/kubernetes-template-project
-  - kubernetes/kube-state-metrics
-  - kubernetes/minikube
-  - kubernetes/node-problem-detector
-  - kubernetes/perf-tests
-  - kubernetes/publishing-bot
-  - kubernetes/release
-  - kubernetes/repo-infra
-  - kubernetes/sig-release
-  - kubernetes/steering
-  - kubernetes/utils
-  - kubernetes-incubator/ip-masq-agent
-  require_self_approval: false
-  lgtm_acts_as_approve: true
-- repos:
-  - kubernetes/kops
-  - kubernetes/kubernetes
-  - kubernetes-client
-  - kubernetes-csi
-  - kubernetes-sigs
-  - client-go/unofficial-docs
-  require_self_approval: false
-- repos:
-  - bazelbuild
-  - kubernetes/community
-  - kubernetes/org
-  - kubernetes/test-infra
-  - kubernetes-sigs/kind
-  require_self_approval: false
-  ignore_review_state: false
-- repos:
-  - helm/charts
-  require_self_approval: false
-  lgtm_acts_as_approve: true
+  orgs:
+    bazelbuild:
+      config:
+        require_self_approval: false
+        ignore_review_state: false
+    client-go:
+      repos:
+        unofficial-docs:
+          config:
+            require_self_approval: false
+    helm:
+      repos:
+        charts:
+          config:
+            require_self_approval: false
+            lgtm_acts_as_approve: true
+    kubernetes:
+      repos:
+        cloud-provider-aws:
+          config:
+            require_self_approval: false
```

Comment:
You can omit these, right? This is the default

Reply:
I wanted to first have a 1to1 mapping of the current config file, which could also be simplified given defaults...

Reply:
Done for require_self_approval and ignore_review_state when they are false.